app/views/notifications/_emergency_banner.html.erb

```diff
@@ -1,9 +1,9 @@
-<section id="emergency-banner-notification" class="<%= banner.campaign_class %>">
+<section id="emergency-banner-notification" class="<%= banner.campaign_class %>" role="banner">
```

Comment:
I think a section would make sense here, but am not sure myself. Regarding the banner role, I just found
> Within any document or application, the author SHOULD mark no more than one element with the banner role.
on https://www.w3.org/TR/wai-aria/roles#banner
But I don't have enough experience to know for sure what the effect would be in assistive technology.